card_reader_server.py

```diff
@@ -116,6 +118,7 @@ def main():
     global keyboard
     global socket_enabled
     global keyboard_enabled
+    global prefix
```

Comment:
should these be instance variables?

Reply:
sorry, file structure was misleading from the diff.  But I do still think that there's a better way than global variables here